packages/flutter/test/material/about_test.dart

```diff
@@ -131,6 +135,72 @@ void main() {
 
     await tester.pumpAndSettle();
 
-    expect(tester.getTopLeft(find.text('DEF')), const Offset(8.0 + safeareaPadding, 527.0));
+    expect(tester.getTopLeft(find.text('DEF')), const Offset(8.0 + safeareaPadding, 287.0));
```

Comment:
This value is changing because before this pull, global state was accumulating between tests.  Now we reset the licenses between each test, so we only show the one this test adds instead of all the other ones before it.